src/SFA.DAS.EAS.Portal.Worker/EventHandlers/PayeScheme/AddedPayeSchemeEventHandler.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Threading.Tasks;
+using NServiceBus;
+using SFA.DAS.EAS.Portal.Application.Commands;
+using SFA.DAS.EAS.Portal.Application.Commands.Paye;
+using SFA.DAS.EmployerAccounts.Messages.Events;
+
+namespace SFA.DAS.EAS.Portal.Worker.EventHandlers.PayeScheme
+{
+    public class AddedPayeSchemeEventHandler : IHandleMessages<AddedPayeSchemeEvent>
```

Comment:
In the CON-409 [PR](https://github.com/SkillsFundingAgency/das-employerapprenticeshipsservice/pull/1646), this handler could be implemented by deriving from `EventHandler`.

Reply:
It could but no more CON-454s wait until one is in master and fix the second one or tech debt it up